xbmc/GUIInfoManager.cpp

```diff
@@ -6762,7 +6762,7 @@ std::string CGUIInfoManager::GetLabel(int info, int contextWindow, std::string *
     {
       std::string linkStatus = g_localizeStrings.Get(151);
       linkStatus += " ";
-      CNetworkInterface* iface = g_application.getNetwork().GetFirstConnectedInterface();
+      CNetworkInterface* iface = g_application.m_ServiceManager->GetNetwork().GetFirstConnectedInterface();
```

Comment:
this should be `CServiceBroker::GetNetwork()` for all calls in this file.

Reply:
done. there is surrounding code using the form i chose earlier..